pallets/pallet-bonded-coins/src/lib.rs

```diff
@@ -269,6 +269,11 @@ pub mod pallet {
 		InvalidInput,
 		Internal,
 		Slippage,
+		/// The requested Operation could not be performed due to insufficient
+		/// balance.
+		InsufficientBalance,
+		/// The calculated collateral is zero.
+		ZeroCollateral,
```

Comment:
what happened to our ZeroAmount error? Is that gone?

Reply:
Yes, I think I deleted that because the check was not really meaningful.